fedn/network/api/server.py

```diff
@@ -636,6 +636,7 @@ def start_server_api():
         if debug:
             app.run(debug=debug, port=port, host=host)
         else:
-            gunicorn_app.run_gunicorn(app)
+            workers=os.cpu_count()
+            gunicorn_app.run_gunicorn(app,host,port,workers)
```

Comment:
I have noticed on many place that you don't add a space after "," in function args, always use a space after , . Will fix so that ruff can do this per auto